src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,32 @@
+@use '../utils/mixins.scss' as *;
+
+.navigation {
+  display: flex;
+  align-items: center;
+  padding: 24px 0 0;
```

Comment:
I'd probably set this property as padding-top for the `header`. 

As a general rule, the parent container should be responsible for positioning it's children. Not a mistake here, but a possibility for cleaner approach.